build-conf/datasets.properties

```diff
@@ -55,4 +55,7 @@ REFERAL=
 SEQAMOD=
 
 # Optional IDz Load Library. Example: FEL.V14R0M0.SFELLOAD
-SFELLOAD=
\ No newline at end of file
+SFELLOAD=
+
+# Optional IDZ zUnit / WAZI VTP library containing necessary copybooks. Example : RDZ.V14R2.SBZUSAMP
```

Comment:
Can you put the same sample dataset as the README.md

Reply:
See recent commit